bql/bql_box_test.go

```diff
@@ -270,7 +270,12 @@ func TestBQLBoxEmitterParams(t *testing.T) {
 		})
 	})
 
-	Convey("Given a BQL statement with an EVERY 1 MILLISECONDS clause", t, func() {
+	// This test is expected "box" receives all tuples generated from "source"
+	// within 1[ms].This expectation is depended on process speed and sometimes
+	// fail.
+	// EVERY expression and semantics are under discussion, so skipped
+	// temporarily.
```

Comment:
`This test expects that "box" receives all tuples generated from "source" within 1ms. Because timing and speed of execution vary every time, this test sometimes fails. To avoid such occasional failures that disturb CI, this test is disabled. Moreover, EVERY is an unofficial feature and will likely get removed in the future relelase. So, skipping this test doesn't affect any official features. This test is just left for a documentation purpose.`